slide/snowflake_retail_lt.md

```
@@ -160,15 +163,42 @@ marp: true
 - IVRの改善と高度化
 
 ---
-
 # お客様相談室の高度化
 
 ## 問い合わせ音声データの分析
 
 - 音声データのテキスト化
-    -　単語の抽出
+    - 単語の抽出
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)